source/slang/slang-ast-support-types.h

```diff
@@ -157,6 +160,9 @@ namespace Slang
         // a vector (this will be added to the cost, if any, of converting
         // the element type of the vector)
         kConversionCost_ScalarToVector = 1,
+        kConversionCost_VectorToScalar = 1,
```

Comment:
Perhaps we should make VectorToScalar smaller than ScalarToVector so we always prefer the overload that makes things simpler. Also might be more accurate to name this OneVectorToScalar.

Reply:
Make sense.
I will rename it and.... raise ScalarToVector by one.